.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,38 @@
+
+name: update-version
+on:
+  pull_request:
+          branches: [master]
+          paths: 
+            - 'version.txt'
+
+jobs:
+  update-version-in-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          ref: ${{ github.head_ref }}
+
+      - if: github.actor == "release-please[bot]"
```

Comment:
Are we sure we only want to bump this version if the actor is release-please? What are the consequences is someone else edits the file?

Reply:
I guess my thought was that nobody should be editing the version file unless they have a really good reason to (in which case they should know to edit the version in both places, and would probably do a global find/replace anyway). I'd personally prefer bots to only push commits on bot PRs.

I'm thinking we could even add a workflow that generates a PR to append the `-dev` to the versions in both places after a new release tag is pushed, so that way we'd never have to manually update versions.

Reply:
That workflow would use GitHub actions, which would use a different user than this does. Does that affect this?

Reply:
nope. because we would be able to update in both places at the same time with the other workflow, so we wouldn't need this to push a commit on to the PR. My thought was that this was more of a configuration that would make the release PRs update in both places since release-please only updates `version.txt` by default.

Reply:
SGTM